intro-to-ruby.rb

```diff
@@ -1,14 +1,24 @@
 #create a string and store it in a variable
-
+y= "chocolate"
+x = "I love #{y} so much."
 #print out the value of the variable you just created
-
+puts x
 #print the sum of 185 and 14
+a= (185)
+b= (14)
 
+puts (a+b)
 #print the product of 1.8 and 3.6
+c= (1.8)
+d= (3.6)
+puts (c*d)
 
 #store the number 78 as a variable
+z= (78)
 
 #print the average of the following numbers: 78, 55, 63, 98
 
+puts ((z+55+63+98)/(4))
```

Comment:
larger parentheses is unnecessary here. Nicely done with substituting in the value.
`puts (z+55+63+98)/4` is sufficient